recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
I think that there has got to be another patch that you can make to point the default location to the installed location.  

Better yet, though, this should be fixed upstream. One of the tenets of conda is that packages are relocatable. If pysdl2 is not relocatable, I am not sure that there is much hope for a conda package. 

Worst case scenario, I think leaving it up to the users to appropriately manage their environment is acceptable. My main issue with the patch here is that it takes that away.  This patch always supercedes what they user says.